batch/batch_test.go

```diff
@@ -2045,7 +2045,7 @@ func mutexClearRegression(t *testing.T, importer featurebase.Importer, sapi feat
 	}
 
 	col := uint64(0)
-	row := uint64(1)
+	row := uint64(0)
```

Comment:
I'm slightly surprised by this change, but on further study, I assume `ineffassign` is allowing ineffective initializations to zero as expressing the type of a newly-created variable, but if you set it to one and then overwrite it before using it, that seems suspicious.